package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "UVL editor",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "rollup -c",
+    "dev": "rollup -c -w"
+  },
+  "dependencies": {
+    "@codemirror/autocomplete": "^6.18.0",
+    "@codemirror/highlight": "^0.19.8",
+    "@codemirror/lang-markdown": "^6.2.5",
+    "@codemirror/language": "^6.10.2",
+    "@codemirror/state": "^6.4.1",
+    "@lezer/highlight": "^1.2.1",
+    "@lezer/lr": "^1.4.2",
+    "@rollup/plugin-node-resolve": "^15.2.3",
```

Comment:
Why are these needed here?

Reply:
Do you mean why i used scoped packages instead of cdn?

Reply:
Packages removed. Those were not necessary